common/app/views/fragments/headerTopNav.scala.html

```diff
@@ -76,7 +77,7 @@
                         <li class="@{(List(" dropdown-menu__item") ++ item.parentClassList).mkString(" ")}">
                             <a class="@{(List(" dropdown-menu__title") ++ item.classList).mkString(" ")}"
                                 @if(item.href.isDefined) { href="@item.href" } @if(item.linkName.isDefined) {
-                                data-link-name="nav3 : topbar : @item.linkName" }>
+                                data-link-name="nav3 : topbar : @item.linkName" } data-link-id="@item.linkId">
```

Comment:
Is this really needed?

Reply:
yes - we use it to select the correct menu item in the js